.run/Compose Deployment.run.xml

```diff
@@ -0,0 +1,11 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
These files should allow sharing configurations for running different scripts/applications from that project in IntelliJ IDEA